index.js

```diff
@@ -522,6 +530,32 @@
         });
       });
 
+  //# reduce :: Foldable f => (a -> b -> a) -> a -> f b -> a
+  //.
+  //. Returns a single value by iterating through a list, successively
+  //. calling an iterator function and passing it an accumulator value and
+  //. the current value from the list, and then passing the result to the
+  //. next call.
+  //.
+  //. ```javascript
+  //. > S.reduce(S.add, 0, [1, 2, 3, 4, 5])
+  //. 15
+  //. ```
```

Comment:
It would be nice to also include an example in which `a` and `b` differ. How about this?

```javascript
> S.reduce((xs, x) => R.concat([x], xs), [], [1, 2, 3, 4, 5])
[5, 4, 3, 2, 1]
```